src/app/content/studyGuides/hooks/locationChange.ts

```diff
@@ -47,6 +47,7 @@ export const hookBody = (services: MiddlewareAPI & AppServices) => async() => {
   const totalCounts = extractTotalCounts(countsPerSource);
 
   services.dispatch(receiveStudyGuidesTotalCounts(totalCounts));
+  services.dispatch(loadMoreStudyGuides());
```

Comment:
@TomWoodward this ensures study guides are populated on load

Reply:
1. when I call it directly it noops because the book hasn't loaded yet
2. if it stops listening to the app locationChange then study guides aren't loaded when the modal is first opened

Reply:
on 2, why does it not get run from here?